docs/zh/deployment/quick-start.md

```diff
@@ -248,3 +248,118 @@ Apollo Config Demo. Please input key to get the value. Input quit to exit.
 app.id=你的appId
 ```
 运行`./demo.sh client`启动Demo客户端即可。
+
```

Comment:
what's the default database for quick start?

Reply:
@nobodyiam I think it will be mysql, since we haven't finish h2 feature.

Reply:
If mysql is the default, then section 5.2 is not necessary?